src/resources/postcss/views/list/_event.pcss

```diff
@@ -30,20 +30,42 @@
 		}
 	}
 
+	.tribe-events-calendar-list__event-row--featured {
+
+		.tribe-events-calendar-list__event-date-tag-datetime {
+			border-left: 3px solid var(--color-accent-primary);
+
+			@media (--viewport-medium) {
+				border-left: none;
+				border-right: 2px solid var(--color-accent-primary);
+				box-shadow: 2px 0 0 0 var(--color-accent-primary);
+			}
+		}
+	}
+
 	.tribe-events-calendar-list__event-date-tag {
 		flex: none;
 		margin-right: var(--grid-gutter-small-half-negative);
-		width: var(--grid-width-1-of-4);
+		width: calc(var(--spacer-9) + var(--grid-gutter-small-half));
```

Comment:
I thought we were going to use no gutters here (removing the `tribe-common-g-row--gutters` class). There's also an overlap that does not allow a user to select half the text, which could be fixed with removing the gutters.
![list-view](https://user-images.githubusercontent.com/16699941/63448878-813b3380-c471-11e9-960b-738c39b2f9bb.png)

Reply:
Yes, the thing is that we need the gutters on desktop. Maybe it's worth to have a partially thing for no gutters on mobile? Like we have with headings?

Reply:
I'd recommend having no gutters and putting in the gutters on desktop.